pio/ppm/ppm.c

```diff
@@ -27,9 +27,9 @@ void set_value_and_log(uint channel, uint value_usec) {
 
 }
 int main() {
-    setup_default_uart();
-    uint pin = 3;
```

Comment:
The file header still notes pin 3 and PIO0, and that should be in sync with the code

Probably it's better to add some macro or a piece of validation code to be sure that PIO, GPIO, and IRQ are in sync